Generative-AI/Ai_voice_agent/app/main.py

```
@@ -0,0 +1,40 @@
+        result = await handle_inbound_call(data)
+        return JSONResponse(content=result)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+@app.post("/outbound-call")
+async def outbound_call(data: dict):
+    try:
+        result = await handle_outbound_call(data)
+        return JSONResponse(content=result)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)